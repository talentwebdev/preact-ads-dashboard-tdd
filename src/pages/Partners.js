import React from 'react'

function Partners() {
    return (
        <div>
            This is a partner pages
        </div>  
    )
}

export default Partners;
