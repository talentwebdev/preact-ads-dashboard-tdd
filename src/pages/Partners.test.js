import { shallow } from "enzyme";
import Partners from "./Partners";

describe("Partners", () => {
  let partnersContainer;

  beforeEach(() => (partnersContainer = shallow(<Partners />)));

  test("should render the div element", () => {
    expect(partnersContainer.find("div")).toBeTruthy();
  });
  it("renders Partners correctly", () => {
    expect(partnersContainer).toMatchSnapshot();
  });
});
