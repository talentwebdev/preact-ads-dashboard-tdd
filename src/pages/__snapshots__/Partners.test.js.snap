// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Partners renders Partners correctly 1`] = `ShallowWrapper {}`;
